src/features/game/mode/move/GameModeMove.ts

```diff
@@ -161,6 +165,24 @@ class GameModeMove implements IGameUI {
     }).setPosition(xPos, yPos);
   }
 
+  /**
+   * Register keyboard listeners for location selection.
+   * Called by activateUI function.
+   */
+  private registerKeyboardListener(): void {
+    const inputManager = GameGlobalAPI.getInstance().getGameManager().getInputManager();
+    const navList2: string[] = this.getLatestNavigations();
+
+    let count = 0;
+    navList2.forEach(nav => {
+      inputManager.registerKeyboardListener(keyboardShortcuts.options[count], 'up', async () => {
+        await GameGlobalAPI.getInstance().swapPhase(GamePhaseType.Sequence);
+        await GameGlobalAPI.getInstance().changeLocationTo(nav);
+      });
+      count += 1;
```

Comment:
You should be able to make use of the in-built indexing for `forEach` instead of declaring a new `count` variable - take a look at the examples [here](https://www.w3schools.com/jsref/jsref_foreach.asp).